### PR TITLE
disable connext testing of client server examples

### PR DESCRIPTION
**Pull request by dhood**
will mention [here](https://github.com/ros2/rmw_connext/issues/168) once merged

### CODE REVIEW

File: rclcpp_examples/CMakeLists.txt [hunk 1]
```diff
@@ -69,17 +69,17 @@ if(BUILD_TESTING)
...
-  set(fastrtps_excluded_example_tests
+  set(opensplice_only_example_tests
     "add_two_ints_server:add_two_ints_client"
     "add_two_ints_server:add_two_ints_client_async")
 
   macro(tests)
     set(example_tests_to_test ${example_tests})
     # TODO(wjwwood): recombine with example_tests this when fastrtps supports wait_for_service.
-    if(rmw_implementation STREQUAL "rmw_fastrtps_cpp")
+    if(NOT rmw_implementation STREQUAL "rmw_opensplice")
```
Comment (dirk-thomas) [inline rclcpp_examples/CMakeLists.txt]: The test looks wrong. Have you tried this code locally or on the build farm? There is not rmw implementation called `rmw_opensplice` - only `rmw_opensplice_cpp`.

Comment (dhood) [inline rclcpp_examples/CMakeLists.txt]: of course - wasn't paying attention to the test count. thank you

**Comment by wjwwood** [comment]
lgtm, I started some before/after CI for this:

- Linux:
  - [Build Status](http://ci.ros2.org/job/ci_linux/1523/) / [Build Status](http://ci.ros2.org/job/ci_linux/1524/)
- OS X:
  - [Build Status](http://ci.ros2.org/job/ci_osx/1193/) / [Build Status](http://ci.ros2.org/job/ci_osx/1194/)

**Comment by wjwwood** [comment]
It seems to remove 14 tests on Linux without adding any warnings or new test failures (that I can see). Does that look right to you @dhood? OS X is still running the second job.

**Comment by dhood** [comment]
I think I'm interpreting the jenkins output differently to you. wasn't it 14 tests that stopped failing?

I expect to see `test_add_two_ints_server_add_two_ints_client_async__rmw_connext_cpp_` and `test_add_two_ints_server_add_two_ints_client_async__rmw_connext_cpp_` removed from http://ci.ros2.org/job/ci_linux/1523/testReport/(root)/ which is happening.  `test_executor__rmw_connext_cpp	` is 'added' (I think because it was a missing result in the 'before' job), so overall the total is one test removed

on OS X the two expected tests are removed

so to me it looks right (as long as I'm right in assuming the executor test change is unrelated)

**Comment by wjwwood** [comment]
Sorry, I was overloading "removed". Seems reasonable to me too. +1 for merge.

**Pull request merged**